src/main/java/com/synopsys/integration/alert/channel/jira/common/util/JiraIssueHandler.java

```diff
@@ -128,14 +106,16 @@ protected String createAdditionalTrackingKey(ComponentItem componentItem) {
         return StringUtils.EMPTY;
     }
 
-    @Override
-    protected String getIssueKey(IssueResponseModel issueModel) {
-        return issueModel.getKey();
+    public Gson getGson() {
+        return gson;
     }
 
-    @Override
-    protected String getIssueTrackerUrl() {
-        return jiraProperties.getUrl();
+    public JiraTransitionHandler getJiraTransitionHelper() {
+        return jiraTransitionHelper;
+    }
+
+    public JiraIssuePropertyHandler getJiraIssuePropertyHelper() {
+        return jiraIssuePropertyHelper;
     }
```

Comment:
Should these be `protected`?

Reply:
Once I made them protected they aren't being used by the sub classes because the subclasses need the concrete class not the abstract base classes the JiraIssueHandler has.  So I removed the get methods for now.